Misc/NEWS.d/next/IDLE/2018-09-22-20-25-07.bpo-34548.7pBzjg.rst

```diff
@@ -0,0 +1 @@
+Make TextView use the text colors from the main IDLE configuration.
```

Comment:
Terry will probably adjust the blurb to be clearer when he merges this (he has uncanny knack for writing succinctly and accurately).  However, I think it should say something more like 'use the colors from the selected configuration theme.'